### PR TITLE
Fix auth errors

### PR DESCRIPTION
**Pull request by paradoux**
- Changed /login handler to set sessionId in cookies instead of returning in body
- Changed endpoint name for signups

### CODE REVIEW

File: packages/server/src/controllers/accounts.ts [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
+    const [accountId] = await createOneAccount(
+      firstName,
+      lastName,
+      email,
+      hashedPassword,
+    );
+
+    const sessionId = await createAccountSession(accountId);
+
+    ctx.cookies.set("laperette_session", sessionId);
```
Comment (AlexGeb) [inline packages/server/src/controllers/accounts.ts:20]: move this cookie name in a constant, as you use it at least 2times

File: packages/server/src/middlewares/authenticate.ts [hunk 2]
```diff
@@ -2,6 +2,16 @@ import { verifySession, verifyPassword } from "../utils/auth";
 import { Context } from "koa";
 import { getAccountByEmail } from "../db/accounts";
 
+export const insertSessionId = (ctx: Context): string | null => {
```
Comment (AlexGeb) [inline packages/server/src/middlewares/authenticate.ts:5]: This insert looks a lot like an extract.. I don’t understand

Comment (AlexGeb) [reply packages/server/src/middlewares/authenticate.ts:5]: Where it is used by the way?

Comment (paradoux) [reply packages/server/src/middlewares/authenticate.ts:5]: Good catch, old attempt not used anywhere :))

File: packages/server/src/controllers/accounts.ts (hunk 1 above)
Comment (paradoux) [reply packages/server/src/controllers/accounts.ts:20]: Yes fair enough, like in an env variable ? Or in a separate file centralizing all the constants?

**Pull request merged**